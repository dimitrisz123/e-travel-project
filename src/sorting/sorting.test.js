import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Sorting from "./sorting";

it("renders Sorting correctly", () => {
	const wrapper = shallow(<Sorting />);
	expect(toJson(wrapper)).toMatchSnapshot();
});
