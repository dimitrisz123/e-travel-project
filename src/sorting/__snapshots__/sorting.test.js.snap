// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Sorting correctly 1`] = `
<select
  className="sorting"
  id="sort-by"
  onChange={[Function]}
>
  <option
    value=""
  >
    Sort By
  </option>
  <option
    value="episode"
  >
    Episode
  </option>
  <option
    value="year"
  >
    Year
  </option>
</select>
`;
